Banyan/src/sessions.jl

```diff
@@ -165,6 +169,7 @@ function start_session(;
         end
     end
     # TODO: Optimize so that we only upload (and download onto cluster) the files if the filename doesn't already exist
+    # Example of f might be "C:/Users/Melany Winston/.../src/clusters.py" --> "cluster.py" extracting out cluster.py
```

Comment:
idk if you want to remove your full name from the example here :)

Reply:
Resolved